src/Oryx.Common/Common.csproj

```diff
@@ -19,6 +19,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.Extensions.Logging" Version="7.0.0" />
     <PackageReference Include="Newtonsoft.Json" Version="13.0.2" />
+    <PackageReference Include="System.Private.Uri" Version="4.3.2" />
```

Comment:
In the end, were you able to validate that if we upgrade xunit to the newest version (2.5.0) in Detector.csproj, it would automatically upgrade System.Private.Uri in the process?
Hopefully Xunit also fixed the vulnerability on their side by upgrading to a newer version of their own dependencies, and as such we might not need to force this version ourselves.
That said, it's not 100% sure they did so it would be good to try to update the nuget and maybe regenerate the deps.json file to see if the wrong System.Private.Uri version is still there?

Reply:
Thanks Dan for the suggestion. I have checked the dependencies of `xUnit 2.5.0`. it is like that:
`xUnit 2.5.0 ---> xunit.assert (>= 2.5.0) ---> NETStandard.Library (>= 1.6.1)  ---> System.Runtime (>= 4.3.0) ---> System.Private.Uri (>= 4.3.0)`
This exactly the same as `xUnit 2.4.2`. That's why I have updated the `System.Private.Uri` version to 4.3.2.